### PR TITLE
Add theme as a locator predicate

### PR DESCRIPTION
**Pull request by caalador**
Enable using theme as a search predicate in locator

### CODE REVIEW

File: karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Locator.kt [hunk 1]
```diff
@@ -42,7 +44,9 @@ public class SearchSpec<T : Component>(
         public var value: Any? = null,
         public var classes: String? = null,
         public var withoutClasses: String? = null,
-        public var predicates: MutableList<Predicate<T>> = mutableListOf()
+        public var predicates: MutableList<Predicate<T>> = mutableListOf(),
```
Comment (mvysny) [inline karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Locator.kt:47]: Awesome idea, thank you :+1: 2 things to fix please:
1. The `var themes` is missing the `public` keyword, to stress that it's a public API
2. The `predicates` list should be the last one, to stress that it's a list of custom stuff of lower importance.

**Review by mvysny — CHANGES_REQUESTED** [review]

File: karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Locator.kt (hunk 1 above)
Comment (caalador) [reply karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Locator.kt]: done, This actually came from Knoobie to the TB project

**Review by mvysny — APPROVED** [review]

**Comment by mvysny** [comment]
@caalador the tests seem to be failing. Could you investigate please?

**Comment by caalador** [comment]
Helps to actually check against the values also.

**Review by mvysny — APPROVED** [review]
looks good, thank you

**Pull request merged**